browser/resources/settings/brave_tor_page/brave_tor_subpage.html

```diff
@@ -0,0 +1,79 @@
+<style include="settings-shared md-select">
+  #bridgesGroup {
+    padding-left: var(--cr-section-padding);
+  }
+
+  #footer {
+    padding-bottom: var(--cr-section-padding);
+  }
+
+  #requestedBridges,
+  #providedBridges {
+    width: 90%;
+    min-height: 70px;
+    height: auto;
+    resize: vertical;
+  }
+
+  textarea {
+    white-space: pre;
+    overflow-wrap: normal;
+    overflow-x: scroll;
+  }
+</style>
+
+<settings-toggle-button id="torEnabled" class="cr-row" pref="[[torEnabledPref_]]" label="$i18n{torEnabledLabel}"
+  sub-label="$i18n{torEnabledDesc}" disabled="[[disableTorOption_]]"
+  on-settings-boolean-control-change="onTorEnabledChange_">
+</settings-toggle-button>
+<settings-toggle-button pref="{{prefs.tor.auto_onion_location}}" class="cr-row" label="$i18n{autoOnionLocationLabel}"
+  sub-label="$i18n{autoOnionLocationDesc}" disabled="[[!torEnabledPref_.value]]">
+</settings-toggle-button>
+
+<settings-toggle-button id="useBridges" class="hr" label="$i18n{torUseBridgesTitle}"
+  sub-label="$i18n{torUseBridgesDesc}" learn-more-url="https://support.brave.com/" checked="{{isUsingBridges_}}"
```

Comment:
Note: this needs a link to the full page.

Reply:
Working on getting the support page live so that we can get a URL for it: https://bravesoftware.slack.com/archives/C02DT9F5EP5/p1658273438421319

Reply:
Here's the full link to the (hidden for now) support page: https://support.brave.com/hc/en-us/articles/7816553516045